events/moose/moose.go

```diff
@@ -573,6 +574,11 @@ func (s *Subscriber) NotifyDisconnect(data events.DataDisconnect) error {
 	if connectionTime <= 0 {
 		connectionTime = -1
 	}
+	// On some arm64 devices connection start time is being reported as maxInt32. This may
```

Comment:
[small] shouldn't it say "connection time" instead of "connection start time" here?